requirements.txt

```
@@ -13,5 +13,5 @@ scikit-fuzzy
 sklearn
 statsmodels
 sympy
-tensorflow; python_version < '3.9'
+tensorflow>=2.5rc2
```

https://github.com/tensorflow/tensorflow/releases
```suggestion
tensorflow>=2.5rc3
```